### PR TITLE
#704 Fix  bug with deserializing model

### PR DESCRIPTION
**Pull request by jojo19893**
*Issue #, if available:*
 #704 
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

Fixing as suggested by @sxjscience .

### CODE REVIEW

**Comment by sxjscience** [comment]
Really appreciate your contribution! Is it possible to also add a test case in the file: https://github.com/awslabs/autogluon/blob/master/text/tests/unittests/test_nlp_text_prediction.py? Basically, you will just need to create a very simple dataset and then run fit + predict.

**Comment by szha** [comment]
Job PR-708-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-708/1/index.html

**Comment by jojo19893** [comment]
> Really appreciate your contribution! Is it possible to also add a test case in the file: https://github.com/awslabs/autogluon/blob/master/text/tests/unittests/test_nlp_text_prediction.py? Basically, you will just need to create a very simple dataset and then run fit + predict.

I added a small test

**Comment by szha** [comment]
Job PR-708-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-708/3/index.html

File: text/tests/unittests/test_nlp_text_prediction.py [hunk 1]
```diff
@@ -178,3 +178,43 @@ def test_mixed_column_type():
...
+                          hyperparameters=test_hyperparameters,
+                          label='score', num_trials=1,
+                          verbosity=4,
+                          ngpus_per_trial=1,
+                          output_directory='./sts_score',
+                          plot_results=False)
+    dev_prediction = predictor1.predict(dev_data)
+    model_path = 'saved_model'
+    predictor1.save(model_path)
+    loaded_predictor = task.load(model_path)
```
Comment (sxjscience) [inline text/tests/unittests/test_nlp_text_prediction.py:218]: You may try to add the `load` + `predict` test to https://github.com/awslabs/autogluon/blob/0dbfe44d075b6f16ffdeae9fb2c812215c227d88/text/tests/unittests/test_nlp_text_prediction.py#L171-L180 ?

Comment (jojo19893) [reply text/tests/unittests/test_nlp_text_prediction.py:183]: sure refactored that :)

**Comment by szha** [comment]
Job PR-708-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-708/4/index.html

**Review by sxjscience — APPROVED** [review]
Thanks for the fix!

**Pull request merged**